Lesson_25/task1.py

```diff
@@ -0,0 +1,195 @@
+"""
+Task 1 - Extend UnorderedList
+Implement append, index, pop, insert methods for UnorderedList.
+Also implement a slice method, which will take two parameters 'start' and
+'stop', and return a copy of the list starting at the position and going up
+to but not including the stop position.
+"""
+
+from node import Node
+
+
+class MyExtendedUnorderedList:
+    """
+    A set of elements, each containing a value and
+    a reference to the next element in the list.
+    The class also contains a reference to the first
+    element of the list. For simplicity, we will assume
+    that lists cannot contain duplicate elements.
+    """
+
+    def __init__(self):
+        self._head = None
+
+    def empty(self):
```

Comment:
`Empty` is both an adjective and a verb in English, it would be better to call it `is_empty`